x-pack/plugins/siem/public/components/event_details/helpers.tsx

```diff
@@ -84,44 +84,47 @@ export const getItems = (data: DetailItem[], id: string): Item[] =>
     description: `${item.description || ''} ${getExampleText(item)}`,
     field: item.field,
     type: item.type,
-    values: item.values.map((itemValue: string) => {
-      const itemDataProvider = {
-        enabled: true,
-        id: escapeDataProviderId(
-          `id-event-field-browser-value-for-${item.field}-${id}-${itemValue}`
-        ),
-        name: item.field,
-        queryMatch: {
-          field: item.field,
-          value: escapeQueryValue(itemValue),
-        },
-        excluded: false,
-        kqlQuery: '',
-        and: [],
-      };
-      return {
-        valueAsString: itemValue,
-        value: (
-          <DraggableWrapper
-            key={`event-field-browser-value-for-${item.field}-${id}-${itemValue}`}
-            dataProvider={itemDataProvider}
-            render={(dataProvider, _, snapshot) =>
-              snapshot.isDragging ? (
-                <DragEffects>
-                  <Provider dataProvider={dataProvider} />
-                </DragEffects>
-              ) : (
-                <FormattedFieldValue
-                  contextId="event-details"
-                  eventId={id}
-                  fieldName={item.field}
-                  fieldType={item.type}
-                  value={parseValue(itemValue)}
+    values:
+      item.values == null
+        ? []
+        : item.values.map((itemValue: string) => {
+            const itemDataProvider = {
+              enabled: true,
+              id: escapeDataProviderId(
+                `id-event-field-browser-value-for-${item.field}-${id}-${itemValue}`
+              ),
+              name: item.field,
+              queryMatch: {
+                field: item.field,
+                value: escapeQueryValue(itemValue),
+              },
+              excluded: false,
+              kqlQuery: '',
+              and: [],
+            };
+            return {
+              valueAsString: itemValue,
+              value: (
+                <DraggableWrapper
+                  key={`event-field-browser-value-for-${item.field}-${id}-${itemValue}`}
+                  dataProvider={itemDataProvider}
+                  render={(dataProvider, _, snapshot) =>
```

Comment:
Would like to know under what situation we name a param after `_`?

Reply:
It is a convention of saying, `_` we have an unused variable placeholder in order to get to the third argument of `snapshot`. This feature is being copied from other programming languages which have it.

So we would name an unused parameter after `_` (if we want there is no rule saying we have to), and if you have two of them it would be `foo(_, __)`

Like in this example I found in our codebase:

```ts
render={(_, __, snapshot) =>
```

In the future TypeScript might support it 🤞 
https://github.com/Microsoft/TypeScript/issues/29202